packages/firestore/lite/src/api/database.ts

```diff
@@ -111,7 +111,6 @@ export class Firestore
    * Only ever called once.
    */
   protected _terminate(): Promise<void> {
-    debugAssert(!this._terminated, 'Cannot invoke _terminate() more than once');
```

Comment:
This doesn't work anymore as this is not called from the AsyncQueue (by the overridden method)